README.md

```diff
@@ -42,6 +42,8 @@ Certain configuration properties MUST be set via enviornment variables. The easi
 | DATA_TRUST_SERVICE | Yes | The base url for the data/trust service server. |
 | BASE_URL | Yes | The base url for this server. |
 | ADMIN_TOKEN | No | The value of the admin token to bypass authorization. If unset no admin token can be used. |
+| AUTH | No | If false or unset, an access token is not required to make requests.
```

Comment:
See comment in other repo but I still think the default (unset) should be to use security (especially since it is part of the spec)